docker/1.4.0/final/Dockerfile.cpu.serving

```diff
@@ -0,0 +1,35 @@
+ARG py_version
+
+FROM awsdeeplearningteam/mxnet-model-server:base-cpu-py$py_version
+
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+
+RUN apt-get update && \
+    apt-get -y install --no-install-recommends \
+    libopencv-dev \
+    build-essential \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /
+
+COPY dist/sagemaker_mxnet_container-2.0.0.tar.gz /sagemaker_mxnet_container-2.0.0.tar.gz
+
+RUN pip install --no-cache mxnet-mkl==1.4.0.post0 \
```

Comment:
Are you going to install sagemaker-inference-toolkit once it is available?

Reply:
sagemaker-inference-toolkit will get installed at the setup.py level within the sagemaker_mxnet_container tarball.